Sources/EpoxyCollectionView/Models/ItemModel/AnyItemModel.swift

```diff
@@ -33,6 +33,10 @@ public struct AnyItemModel: EpoxyModeled {
 
 }
 
+// MARK: ErasedContentProviding
+
+extension AnyItemModel: ErasedContentProviding {}
```

Comment:
Mimics this conformance: https://github.com/airbnb/epoxy-ios/blob/e46fab4024d4e8e0d5754d7add8be1ccb94390f3/Sources/EpoxyCollectionView/Models/ItemModel/ItemModel.swift#L121-L123